docs/snapshotters/README.md

```diff
@@ -62,3 +62,58 @@ However, the mountpoint `/path/inside/container` needs to exist for the bind
 mount, so one of the previous mounts must be responsible for providing that
 directory in the rootfs. In this case, one of the lower dirs of the overlay has
 that directory to enable the bind mount.
+
+## Multiple snapshotters via runtime handlers
```

Comment:
This should be moved to https://github.com/containerd/containerd/tree/main/docs/cri , as the concept of the runtime handler is specific to CRI.

Can be still linked from here